support-frontend/assets/components/subscriptionCheckouts/address/addressFieldsStore.ts

```diff
@@ -236,11 +236,13 @@ const setFormErrorsFor =
 const addressActionCreatorsFor = (scope: AddressType) => ({
 	setCountry:
 		(countryRaw: string) =>
-		(dispatch: Dispatch<SetCountryChangedAction | SetCountryAction>) => {
+		(
+			dispatch: Dispatch<SetCountryChangedAction | PayloadAction<IsoCountry>>,
```

Comment:
When we're further along with the migration we'll be able to get a `Dispatch` type inferred out of the store type, but for the time being in order to harmonise with our existing action types I've had to use the Redux Toolkit `PayloadAction` type explicitly in a few places.